lua/copilot_cmp/init.lua

```diff
@@ -17,7 +17,7 @@ local default_opts = {
 M._on_insert_enter = function(opts)
 
   local find_buf_client = function()
-    for _, client in ipairs(vim.lsp.get_active_clients()) do
+    for _, client in ipairs(vim.lsp.get_clients()) do
```

Comment:
To preserve compatibility with older versions while avoiding the deprecation warning:
```suggestion
    local get_clients = vim.lsp.get_clients or vim.lsp_get_active_clients
    for _, client in ipairs(clients) do
```